cmd/inspector/main.go

```diff
@@ -472,20 +476,73 @@ func ObjectHealth(cmd *cobra.Command, args []string) (err error) {
 		return ErrArgs.Wrap(err)
 	}
 
+	startAfterSegment := int64(0) // start from first segment
```

Comment:
this file is definitely too large

Reply:
you mean inspector/main.go has too many lines?